Changelog.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
-## Unreleased
+## 2004
```

Comment:
is that year 20(2) month 04 ?

Reply:
@tgeens Yes. This formatting is rather based on Microsoft Windows 10 versioning.
I was thinking to follow Ubuntu's version convention like 20.04.
I wanted to change it for docker-openjdk but you already merged the PR so I kept the same release naming here.
Ubuntu's cloud image releases follow a full date convention like 20200419.
For clarity we could also add Version or Release as prefix.
I agree seeing 2004 as a release is a bit odd.

Reply:
@tgoons We also could use a versioning scheme like yy.mm.x.
Example: 20.04.0, 20.04.1, 20.04.2, ... 

x would be incremented for each needed release in that month and be reset at the start of another month. In that way we adhere to the 3-part versioning scheme (major.minor.micro) we use for our applications. We can also add then a release date in the changelog.

What do you think?

Reply:
- Do we want to increase the version number for an OS update, as in: upstream OS image updated ?
- I don't want to hold up rebuilds by not doing a _release_ automatically. Which means every new build is a release (and possibly version bump ?)

Reply:
1. I do think it is a good idea to increase the version number for an OS update as well. But to avoid all the overhead triggering a new build when the OS base images is update would do for now.
2. I understand and I agree. I think using a full date version "yyyymmdd" would allow to automate the versioning 

I propose we further discuss this topic in the AB meeting.

Will merge the changes for now.